src/pilot/templates/dell-environment.yaml

```diff
@@ -44,6 +55,13 @@ parameter_defaults:
 
   # Additional hiera data for all nodes
   ExtraConfig:
+    # NFV Profile settings start here
+    dellnfv::hugepages::enable: True
+    dellnfv::hugepages::hugepagesize: 2M
+    dellnfv::hugepages::hugepagecount: 49152
+    dellnfv::numa::enable: True
+    dellnfv::numa::vcpu_pin_set: \"8-47\"
+
```

Comment:
The above dellnfv::hugepages::enable & dellnfv::numa::enable should be set to False since XSP is our default profile.